gradle.properties

```diff
@@ -1,4 +1,4 @@
 group = com.enonic.lib
 projectName = lib-thymeleaf
-xpVersion = 7.13.0
-version=2.1.2-SNAPSHOT
+xpVersion = 8.0.0-SNAPSHOT
```

Comment:
Can we implement it without 8.0?

Reply:
Yes, we can. I thought we do it in scope XP 8.

Reply:
Sounds strange but we better support xp7, too